source/merge-deep.d.ts

```diff
@@ -0,0 +1,136 @@
+interface MergeDeepOptions {
+  strict?: boolean;
+}
+
+type UnknownRecord = Record<string, unknown>;
```

Comment:
Shouldn't this be:

```suggestion
type UnknownRecord = Record<PropertyKey, unknown>;
```

?

Reply:
Any maybe it should be named `AnyRecord` and moved to internal.d.ts?